tfjs-backend-webgpu/src/buffer_manager.ts

```diff
@@ -84,16 +86,9 @@ export class BufferManager {
     this.numBytesUsed -= size;
   }
 
-  releaseUploadBuffer(
-      buffer: GPUBuffer, size: number, usage: GPUBufferUsageFlags) {
-    buffer.mapAsync(GPUMapMode.WRITE)
-        .then(
-            () => {
-              this.releaseBuffer(buffer, size, usage);
-            },
-            (err) => {
-                // Do nothing;
-            });
+  releaseUploadBuffer(buffer: GPUBuffer) {
```

Comment:
We can also remove this. But maybe we need a function BufferManager.dispose(buffer) to cover.